packages/portfolio-contract/test/network/prod-network.test.ts

```diff
@@ -0,0 +1,151 @@
+import test from 'ava';
+import { Far } from '@endo/marshal';
+
+import { makeGraphFromDefinition } from '../../tools/network/buildGraph.js';
+import PROD_NETWORK, {
+  PROD_NETWORK as NAMED_PROD,
+} from '../../tools/network/network.prod.js';
+import type { AssetPlaceRef } from '../../src/type-guards-steps.js';
+
+const brand = Far('TestBrand') as any;
+const feeBrand = Far('TestFeeBrand') as any;
+
+const toSet = <T>(iter: Iterable<T>) => new Set(iter);
+
+// Shared expectations
+const HUBS = [
+  '@agoric',
+  '@noble',
+  '@Arbitrum',
+  '@Avalanche',
+  '@Base',
+  '@Ethereum',
+  '@Optimism',
+];
```

Comment:
Make sure this matches `SupportedChain`.

Either by construction...

```suggestion
const HUBS = SupportedChain.map(c => `@${c}`);
```

or by a test that checks consistency.

Reply:
I'll add a test for consistency. That was the test remains a direct test of requirements.